examples/kill-namespace.js

```diff
@@ -0,0 +1,40 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { KubernetesChaosClient } from '../src/chaos.js';
+
+function getNamespaceYaml(name) {
+    return `kind: Namespace
+apiVersion: v1
+metadata:
+  name: ` + name + `
+  labels:
+    name: ` + name + `
+`;
+}
+
+export default function () {
+
+  // KubernetesChaosClient requires the Kubernetes client from xk6-kubernetes.
+  const k8sClient = new Kubernetes({})
+
+  // create a namespace
+  const namespaceName = "namespace-name";
+  k8sClient.namespaces.apply(getNamespaceYaml(namespaceName));
```

Comment:
Can we move this helper to xk6-kubernetes? I have the feeling that creating a namespace, should be easier than having to use Yaml. Something like: ```k8sClient.namespaces.Create(`name`)```

Reply:
That functionality is already available:
```
  k8cClient.namespaces.create({
    name: namespace
  })
```

Reply:
Oh, if that's the case, even better! We can use it :D